### PR TITLE
Stop using _openExistential

### PR DESCRIPTION
**Pull request by rintaro**
[SE-0352 Implicitly Opened Existential](https://github.com/apple/swift-evolution/blob/main/proposals/0352-implicit-open-existentials.md) is supported widely enough.

### CODE REVIEW

File: Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift [hunk 1]
```diff
@@ -14,26 +14,6 @@ import SwiftDiagnostics
...
-}
-
-private func expandConformanceMacro(attribute: AttributeSyntax, attachedTo: DeclSyntax) -> CodeBlockItemListSyntax {
-  fatalError("unimplemented")
-}
-
-private func expandAccessorMacro(attribute: AttributeSyntax, attachedTo: DeclSyntax) -> AccessorListSyntax {
-  fatalError("unimplemented")
-}
-
```
Comment (rintaro) [inline Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift:36]: This part was added unintentionally when I copied MacroSystem in #1858

File: Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift [hunk 2]
```diff
@@ -493,27 +470,6 @@ extension MacroApplication {
     )
   }
 
-  private func expandMemberAttribute(
```
Comment (rintaro) [inline Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift:496]: This function was not used.

**Comment by rintaro** [comment]
@swift-ci Please test

**Review by bnbarham — APPROVED** [review]

**Comment by rintaro** [comment]
@swift-ci Please test

**Comment by rintaro** [comment]
@swift-ci Please test Windows

**Pull request merged**